app/assets/stylesheets/layouts/_main-content.scss

```diff
@@ -0,0 +1,7 @@
+.main-content {
+  position: relative;
+  top: 0px;
+  left: 25%;
```

Comment:
This now depends on the width. If you want to centre this container you might just want `auto` on the left and right.